css/css-scoping/has-slotted-001.tentative.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+<meta charset="utf-8">
+<title>:has-slotted(*)</title>
+<link rel="help" href="https://github.com/w3c/csswg-drafts/pull/10586">
+<link rel="match" href="/css/reference/ref-filled-green-100px-square-only.html">
+<p>Test passes if there is a filled green square.</p>
+<div id="host">
+    <template shadowrootmode="open">
```

Comment:
I think these need to be re-indented with tabs to pass lint.

Reply:
🙈 Every time!